packages/state-manager/src/sagas/publicChannels/publicChannels.slice.ts

```diff
@@ -29,24 +33,44 @@ const log = logger('publicChannels')
 
 export class PublicChannelsState {
   public currentChannelAddress: string = 'general'
+  public isGeneralRecreation: boolean = false
```

Comment:
Once again it's just a minor naming thing, but 'pendingGeneralChannelRecreation' would be more accurate, in my opinion

Reply:
+1